### PR TITLE
Print out list of modified issues, it's useful for when things go wro…

### PR DESCRIPTION
**Pull request by estellecomment**
…ng and you have to fix them.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -124,6 +124,8 @@ if (dryRun) {
 const cutoffDate = moment().subtract(numDaysOld, 'd').format('YYYY-MM-DD');
 console.log(`Getting issues that haven't been touched since ${cutoffDate}`);
 
+var pickedIssues;
```
Comment (SCdF) [inline index.js]: const plz

**Review by SCdF — CHANGES_REQUESTED** [review]
This isn't a list of the issues you've modified. This is the list of issues you *might* modify. This list gets shuffled, then some of the items in the list get assigned to people, but not necessarily all.

Also, we already output what we are doing to every issue. If you're looking to compact that to just be the list of issues not the actual interactions I imagine you can just do that.

If you want to keep the more descriptive logging as well add a `--verbose` or `--terse` mode, default one or the other and put the other behind that flag.

File: index.js (hunk 1 above)
Comment (estellecomment) [reply index.js]: can't, I'm reassigning it later on. 
I'm assigning a value in one promise, and then printing it out in another, so I need it declared outside of the whole promise chain. 
If there's a better way to pass it around I'm happy to do that.

**Comment by estellecomment** [comment]
Hopefully this one is actually all ES6 :)

Added verbose mode (default : true).
Example of non-verbose output : 
```
Dry-run enabled!
Found 52 un-dealt-with issues in medic/medic-webapp
DRYRUN, NO ACTION TAKEN - 2224 assigned to estellecomment, labeled, commented.
DRYRUN, NO ACTION TAKEN - 3057 assigned to estellecomment, labeled, commented.
DRYRUN, NO ACTION TAKEN - 2504 assigned to estellecomment, labeled, commented.
DRYRUN, NO ACTION TAKEN - 1898 assigned to estellecomment, labeled, commented.
All done!
```
Verbose output has all the things, including the list of possibly-modified issues at the end. (I'm printing them out whether we successfully modified them or not, on purpose)

File: index.js (hunk 1 above)
Comment (SCdF) [reply index.js]: @estellecomment OK then use `let`, that is `var` but with correct scoping rules. There should be almost no need to ever use `var` anymore :-)

Comment (SCdF) [reply index.js]: (I see in a later commit you changed it to `const pickedIssues = [];`, that works too.

**Comment by SCdF** [comment]
Looks good.

> I'm printing them out whether we successfully modified them or not, on purpose)

Yep that's cool, in this version it's fine. To be clear about what I meant before, in this version `pickedIssues` is a list of issues that had changes attempted on them, whereas in the previous cut of this it was just all issues that were pulled from github, regardless of whether they were going to be changed or not.

**Review by SCdF — APPROVED** [review]
Approved. I'll leave you to merge to get the commit message how you like, with additional stuff about verbose etc.

**Pull request merged**